ops/trigger_http.go

```diff
@@ -77,6 +77,7 @@ func remoteBackupTrigger(args []string) error {
 	}
 
 	log.Println(string(body))
+	log.Println(resp.Header)
```

Comment:
Made the changes. Any specific reasons for not using `log.Println()` and and going with `fmt.Printf()` ?